pkg/api/apiloader_test.go

```diff
@@ -398,6 +405,45 @@ func TestLoadContainerServiceWithEmptyLocationCustomCloud(t *testing.T) {
 	}
 }
 
+func TestDeserializeContainerService(t *testing.T) {
+	apiloader := &Apiloader{
+		Translator: nil,
+	}
+
+	// Test AKS Engine api model
+	cs, version, err := apiloader.DeserializeContainerService([]byte(exampleAPIModel), false, false, nil)
+	if err != nil {
+		t.Fatalf("unexpected error deserializing the example apimodel: %s", err)
```

Comment:
Any specific reason for using `t.Fatalf`?

Reply:
copy/pasta, will change